### ISSUE TITLE
issue running with kubernetes master (1.13)

### ISSUE THREAD

**Issue by Huang-Wei** [kubernetes-sigs/kubeadm-dind-cluster#223]
It seems in latest kubernetes master branch (1.13), kubeadm API is promoted to `kubeadm.k8s.io/v1alpha3`, and it brings some breaking changes.

So that `image/kubeadm.conf.tmpl` should be updated to: (not sure if it's 100% correct)

```yaml
# Customize this, as needed, for your situation, but keep
# the variables defined herein:
#   ADV_ADDR, POD_SUBNET_DISABLE, POD_NETWORK_CIDR,
#   SVC_SUBNET, BIND_ADDR
apiVersion: {{API_VERSION}}
kind: ClusterConfiguration
kubernetesVersion: "{{KUBEADM_VERSION}}"
networking:
  {{POD_SUBNET_DISABLE}}podSubnet: "{{POD_NETWORK_CIDR}}"
  serviceSubnet: "{{SVC_SUBNET}}"
unifiedControlPlaneImage: mirantis/hypokube:final
apiServerExtraArgs:
  insecure-bind-address: "{{BIND_ADDR}}"
  insecure-port: "{{BIND_PORT}}"
  {{COMPONENT_FEATURE_GATES}}
{{APISERVER_EXTRA_ARGS}
controllerManagerExtraArgs:
  {{COMPONENT_FEATURE_GATES}}
{{CONTROLLER_MANAGER_EXTRA_ARGS}}
schedulerExtraArgs:
  {{COMPONENT_FEATURE_GATES}}
{{SCHEDULER_EXTRA_ARGS}}
featureGates: {{FEATURE_GATES}}
---
apiEndpoint:
  advertiseAddress: "{{ADV_ADDR}}"
apiVersion: {{API_VERSION}}
kind: InitConfiguration
```

And modifying this file locally doesn't help, I guess this should be packaged as a new `mirantis/kubeadm-dind-cluster` version.

**Comment by rpothier** [comment]
@Huang-Wei can you try the above fix?

**Comment by Huang-Wei** [comment]
@rpothier nope, it doesn't.

```console
[config] WARNING: Ignored YAML document with GroupVersionKind kubeadm.k8s.io/v1alpha3, Kind=MasterConfiguration
no InitConfiguration or ClusterConfiguration kind was found in the YAML file
```

As i mentioned earlier, I guess it's because `image/kubeadm.conf.tmpl` is packaged into the image `mirantis/kubeadm-dind-cluster`. So modifying the one outside doesn't help.

**Comment by rpothier** [comment]
@Huang-Wei  The image needs to be rebuilt. From the kubeadm-dind-cluster directory you can build with ./build/build-local.sh.
Then set this environmental variable  in the shell that you do the cluster up
DIND_IMAGE=mirantis/kubeadm-dind-cluster:local

**Comment by Huang-Wei** [comment]
@rpothier thanks! It works perfectly with rebuilt image.

**Comment by Huang-Wei** [comment]
This issue has been fixed in #226. Close it.
